### DISCUSSION TITLE
Any way to implement this with SliverAppBar?

### DISCUSSION THREAD

**Discussion by daniel-forsdike** [fe-rai/scrollable_list_tab_scroller#3] [Q&A]
Hi @railson-ferreira, looks like a great package thanks so much for your hard work. Do you have any ideas on the best way to implement this package with https://api.flutter.dev/flutter/material/SliverAppBar-class.html

Many thanks.

**Comment by aleksandar-radivojevic** [comment]
Any updates on this?

**Comment by fe-rai** [comment]
Sorry for the delay. This widget use as a dependency 'scrollable_positioned_list', I was wondering about the right way to implement it in a custom scroll. I haven't come up with a good solution for this, thinking about keeping the high level of customization.
